Userland/Libraries/LibGUI/MessageBox.cpp

```diff
@@ -55,6 +55,25 @@ int MessageBox::ask_about_unsaved_changes(Window* parent_window, StringView path
     return box->exec();
 }
 
+int MessageBox::ask_about_revert_file(Window* parent_window, StringView path)
```

Comment:
This name feels really cryptic. What we're really doing is asking about reloading a file that has changed on disk.
`MessageBox::ask_to_reload_file_that_has_changed_on_disk()` or something like that perhaps?

Reply:
Done!